src/docfx/build/xref/XrefResolver.cs

```diff
@@ -166,7 +172,7 @@ public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
                     .ToArray();
             }
 
-            var model = new XrefMapModel { References = references };
+            var model = new XrefMapModel { References = references, RepositoryName = _config.Name };
```

Comment:
`_config.Name` is not the repository name, it is the docset name. To get the repository url, use `_repository?.Remote`.

Reply:
OK